doc/src/learn/single-writer-apps.md

```diff
@@ -21,6 +21,7 @@ final step and not for every action, such as voting and lotteries and DeFi Oracl
 1. Personal GitHub, Overleaf LaTex editor, wish/shopping lists, etc.
 1. Personal password manager.
 1. Non-interactive games (i.e., advertise/evolve your SimCity, FarmVille state etc.).
+1. Human vs. Computer games (i.e., chess AI that is programmed into the smart contract. The AI automatically plays back in the same transaction of user's chess move).
```

Comment:
The AI automatically plays back in the same ... of user's chess move.
Does this mean that the AI takes its turn and its move is performed in the same transaction? This line could be a bit difficult to understand for ESL folks.

Reply:
Yes you play a move by submitting a tx, and the smart contract will execute your move + play AI's move. Then the user can see how the chess state has been updated by both moves (both were registered in the above tx).